results.tex

```diff
@@ -5,6 +5,42 @@ \section{Results}
 This section describes each of these features in sufficient detail in the
 following text.
 
+\begin{figure}[htb]
+\centering
+  \begin{subfigure}[b]{.35\columnwidth}
+    \centering
+    \includegraphics[width=\columnwidth]{tooth_front_.png}
+    \caption{Volume only.}
+    \label{fig:tooth_front}
+  \end{subfigure}
+  \begin{subfigure}[b]{.35\columnwidth}
+    \centering
+    \includegraphics[width=\columnwidth]{tooth_front_enamel.png}
+    \caption{Intermixed.}
+    \label{fig:tooth_front_enamel}
+  \end{subfigure}
+  \begin{subfigure}[b]{.35\columnwidth}
+    \centering
+    \includegraphics[width=\columnwidth]{tooth_bottom_.png}
+    \caption{Volume only \\(rotated).}
+    \label{fig:tooth_bottom}
+  \end{subfigure}
+  \begin{subfigure}[b]{.35\columnwidth}
+    \centering
+    \includegraphics[width=\columnwidth]{tooth_bottom_enamel.png}
+    \caption{Intermixed \\(rotated).}
+    \label{fig:tooth_bottom_enamel}
+  \end{subfigure}
+  \caption{Industrial CT scan of a human tooth from ~\citep{pfister_tf_bakeoff_2001}.
+    The root and dentin of the tooth are rendered as a volume
+    (~\Autoref{fig:tooth_front}) while an iso-contour of the enamel is rendered
+    as translucent polygonal geometry (~\Autoref{fig:tooth_front_enamel}).
+    ~\Autoref{fig:tooth_bottom} and ~\Autoref{fig:tooth_bottom_enamel} show a
```

Comment:
Thanks @sankhesh , I did not know this was the case, saw the extra spaces but I forgot to fix them.